src/JsonSchema/Constraints/UndefinedConstraint.php

```diff
@@ -169,7 +169,7 @@ protected function validateCommonProperties(&$value, $schema = null, JsonPointer
             } elseif (isset($schema->required) && !is_array($schema->required)) {
                 // Draft 3 - Required attribute - e.g. "foo": {"type": "string", "required": true}
                 if ($schema->required && $value instanceof self) {
-                    $this->addError($path, 'Is missing and it is required', 'required');
+                    $this->addError($path, "The property " . $schema->name . " is required", 'required');
```

Comment:
`$schema->name` does not exist, and causes test failures.
Noting this is the draft-03 boolean require you have added this for, the property name will not be available via `$schema` - you will need to obtain it from elsewhere.